cmd/build/command.go

```diff
@@ -202,7 +203,9 @@ func validateDockerfile(file string) error {
 		return err
 	}
 
-	_, _, err = instructions.Parse(parsedDockerfile.AST)
+	_, _, err = instructions.Parse(parsedDockerfile.AST, linter.New(&linter.Config{
```

Comment:
Could this have some side effect? Was this having the same default value before?

Reply:
It's having the same effect as before. Tested on scenario 1